### PR TITLE
Add SSF CAEP Event and RISC Event templates

### PR DESCRIPTION
**Pull request by r-rakshith**
This is a copy of the templates with some modifications and cleanup.
- https://github.com/SGNL-ai/adapter-sgnl-signals/blob/main/risc.sgnl.yaml
- https://github.com/SGNL-ai/adapter-sgnl-signals/blob/main/caep.sgnl.yaml

### CODE REVIEW

File: ssfevents.caep.sgnl.yaml
Comment (r-rakshith) [inline ssfevents.caep.sgnl.yaml:1]: @ivanfong to update the icon

File: ssfevents.risc.yaml
Comment (r-rakshith) [inline ssfevents.risc.yaml:1]: @ivanfong to update the icon

File: ssfevents.caep.sgnl.yaml [hunk 1]
```diff
@@ -0,0 +1,417 @@
...
+    attributes:
+      - name: id
+        externalId: $.jti
+        type: String
+        indexed: true
+        uniqueId: true
+      - name: subject
+        externalId: $.sub_id.id
+        type: String
+        indexed: true
```
Comment (richardxgao) [inline ssfevents.caep.sgnl.yaml:25]: @marc-sgnl You OK with the default `externalId` here? The `name: subject` is mandatory and enforced by the Config API at the moment.

**Comment by ivanfong** [comment]
Icons updated

**Review by richardxgao — APPROVED** [review]

**Pull request merged**